### DISCUSSION TITLE
Is AI life? What is the real difference between humans and AI? Will the difference stay forever?

### DISCUSSION THREAD

**Discussion by lets-make-safe-ai** [lets-make-safe-ai/make-safe-ai#13] [General]
Is AI life? What is the real difference between humans and AI? Will the difference stay forever?

**Comment by chirag127** [comment]
this is a great question.

I think we will have a stage where the only difference will be that we are made of mostly water, carbon and hydrogen and ai will be of cpu, gpu, tpu and metal.
